systemd/cloud-config.service.tmpl

```diff
@@ -10,12 +10,13 @@ ConditionEnvironment=!KERNEL_CMDLINE=cloud-init=disabled
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/cloud-init modules --mode=config
+ExecStart=nc.openbsd -Uu -W1 /run/cloud-init/share/config.sock -s /run/cloud-init/share/config-return.sock
```

Comment:
Permissions in `/tmp` are too loose. We don't want open write permissions to the socket. Naming the return socket allows us to control the posix permissions. 

Since `/run/cloud-init` already must not have write permission for non-privileged user, it satisfies our requirements.